CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 5.5.1-SNAPSHOT (future release)
```

Comment:
New feature is added => minor version bump here and in pom.xml files: (5.6.0-SNAPSHOT)